resources/assets/js/components/ProjectDetailsComponent.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div class="header">
+
+    <div class="card-body">
+      <div class="project-info container">
+        <div class="head-section row">
+          <h3 class="col-sm-12"><strong>{{project.name}}</strong></h3>
+        </div>
+
+        <div class="info-section row">
+          <div class="current-cycle-info col-sm-6">
+            <label for=""><strong>Current Cycle:</strong></label>
+            {{"29/10/2018 to 02/10/2018"}}
+            <br>
+            <label for=""><strong>Estimated Effort:</strong></label>
+            {{"40"}}
+            <br>
+            <label for=""><strong>Last Updated:</strong></label>
+            {{"30/10/2018"}}
+          </div>
+          <div class="employee-info col-sm-6">
+            <div class="row list-section">
+              <div class="offset-sm-4 col-sm-4 list-section">
+                <p><strong>Employees</strong></p>
+                <ul class="unstyled-list">
+                  <li v-for="employee in project.employees">{{employee.name}}</li>
+                </ul>
+              </div>
+              <div class="col-sm-4 list-section">
+                <p><strong>Contribution</strong></p>
+                <ul class="unstyled-list">
+                  <li v-for="employee in project.employees">0</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <hr>
+      <div class="project-actions row">
+        <div class="col-sm-12 project-action">
+          <table class="table table-striped table-bordered">
+            <thead>
+              <tr>
+                <th>
+                  Employee
+                  <span @click="showAddEmployeeForm">
+                    <i class="btn btn-default fa fa-plus"></i>
```

Comment:
Not sure how this is showing up but I'll prefer a cleaner structure,
```suggestion
<button type="button" class="btn btn-default">
    <i class="fa fa-plus"></i>
</button>
```